File: official/docs/java/v6/child-users/delete.java
package users;

import com.easypost.exception.EasyPostException;
import com.easypost.service.EasyPostClient;

public class Delete {
    public static void main(String[] args) throws EasyPostException {
        EasyPostClient client = new EasyPostClient(System.getenv("EASYPOST_API_KEY"));

        client.user.delete("user_...");
    }
}

File: official/docs/golang/current/child-users/delete.go
package example

import (
	"fmt"

	"github.com/EasyPost/easypost-go/v5"
)

func delete() {
	client := easypost.New("EASYPOST_API_KEY")

	err := client.DeleteUser("user_...")

	fmt.Println(err)
}
